google/cloud/storage/client.h

```diff
@@ -427,12 +475,18 @@ class Client {
    * @param options a list of optional query parameters and/or request headers.
    *     Valid types for this operation include `UserProject`.
    *
-   * @throw std::runtime_error if the operation fails.
+   * @throw std::runtime_error if there is a permanent failure, or if there were
+   *     more transient failures than allowed by the current retry policy.
    *
-   * @par Examples
+   * @par Idempotency
+   * Like other operations that mutate the state of GCS, this operation is only
+   * idempotent if restricted by pre-conditions, in this case,
+   * `IfMetagenerationMatch`.
```

Comment:
I don't think `Like other operations that mutate the state of GCS` is necessary. 

> This operation is only idempotent if restricted by pre-conditions, in this case, `IfMetagenerationMatch`.

Please apply to related comments.

Reply:
Done.